resources/state_of_the_dapps.json

```diff
@@ -0,0 +1,43 @@
+{
+    "auction-house":
+    {
+        "name":
+        {
+            "en": "Auction House"
+        },
+        "photo-path": "contacts://auction-house",
+        "state-of-the-dapp?": true,
```

Comment:
Agree
Don't like the localized name as a group name because it can be a bit misleading for our translators. Maybe better to use `"dapp-group": "state-of-the-dapps"` or `"dapp-group": "dapps"` and use this property only if `dapp?` is true